taker/src/routes.rs

```diff
@@ -336,6 +340,15 @@ pub async fn get_version() -> Json<HealthCheck> {
     })
 }
 
+#[rocket::get("/seed_backup")]
+#[instrument(name = "GET /seed_backup", skip_all)]
+pub async fn get_seed_backup(
+    seed: &State<Arc<ThreadSafeSeed>>,
```

Comment:
Don't forget to add the `user` guard, otherwise this API does not need authentication.